storage/posix/files.go

```diff
@@ -128,8 +128,19 @@ func (s *Storage) unlockCP() error {
 	return nil
 }
 
-// Add commits to sequence numbers for an entry
-// Returns the sequence number assigned to the first entry in the batch, or an error.
+// Add takes an entry and queues it for inclusion in the log.
+// Upon storing the entry in a queue to be sequenced, it returns a future that will
+// evaluate to either the sequence number assigned to this entry, or an error.
+// This future is made available when the entry is queued, but not sequenced.
```

Comment:
The last sentence of this para is correct, but might be easy to misinterpret. At the risk of a little repeating, how about "The future is returned/made available when the entry is queued, but not fulfilled until the entry is durably sequenced"?

Reply:
Reworked this. PTAL?

Reply:
Bonza